lib/src/icons/Carat-Up.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import './Carat_Up.scss';
+
+const CaratUp = () =>
+  <svg viewBox="0 0 12 8" className="nx-icon nx-icon--carat-up">
+    <path fill="currentColor"
+          fill-rule="nonzero"
```

Comment:
Another fill-rule to get rid of

Reply:
> Another fill-rule to get rid of

https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/01f6164604ec672486d935eb13c9eb4c3e460364